features/data-layer-migration.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Data Layer Migration
+
+  Background:
+    Given I haven't clicked the migration button
+    And I have completed the basic setup
```

Comment:
@daniloprates let's actually add the "acceptance of the data layer migration" to the basic setup: https://github.com/input-output-hk/daedalus/blob/feature/ddw-417-implement-new-data-layer-migration-screen/features/step_definitions/setup-steps.js#L5
Then here in the `Background` steps of these feature we should do the same as in the same part of the `Accept the terms of use` feature - manually select language and accept the terms. This would eliminate the need of having to run the acceptance of the data layer migration in the scope of the CreateWallets step definition!